docs/source/conda-package-walkthrough/conda_install.rst

```diff
@@ -0,0 +1,54 @@
+*********************
+Installing with conda
+*********************
+
+Conda install
+-------------
+
+* When you `conda install <package>` the happy path (with no dependencies)
```

Comment:
I think it would be neat if this turns into a flow chart or something at some point and also includes not just the "happy path". Maybe that can be for another PR though?

Reply:
Agreed that this would be a nice option for another PR. I made at ticket for it: https://anaconda.atlassian.net/browse/DSNC-972